[CS2113-T14-1] History Flashcards

Zhilin-Huang opened the pull request:
@g-lilian 
@sliu107 
@quinnyyy 
@Zhilin-Huang

g-lilian committed:
Add JUnit tests for ListFlashcardsInGroupCommand and ShowGroupsCommand, update TimelineCommandTest

sliu107 committed:
add random review feature in DG

sliu107 committed:
Merge pull request #118 from g-lilian/master

Add JUnit tests for ListFlashcardsInGroupCommand and ShowGroupsCommand, update TimelineCommandTest

sliu107 committed:
Merge pull request #119 from sliu107/random-review-feature-in-DG

add random review feature in DG

sliu107 committed:
Merge pull request #117 from g-lilian/developerguide

Add set priority feature implementation to DG

Zhilin-Huang committed:
Update sequence diagrams for get and update study plan

Zhilin-Huang committed:
Update code formatting

Zhilin-Huang committed:
Remove unnecessary semicolons

g-lilian committed:
Edit help message

Included all commands thus far and reorganized into categories. Used [] to indicate parameters/arguments.

g-lilian committed:
Merge pull request #127 from g-lilian/master

Update help message

g-lilian committed:
Create separate ui function for ListReviewedCommand

Used separate function to make the ui messages more specific to the command, in case it looks like a bug.

g-lilian committed:
Add confirmation message for study plan updating

g-lilian committed:
Update Ui messages to be more accurate

g-lilian committed:
Update text-ui-test

g-lilian committed:
Update tests based on changes to Ui

g-lilian committed:
Add latest commands to UG

Added skeleton of latest commands to UG, also modified the command format section to reflect our current input format.

g-lilian committed:
Merge pull request #129 from g-lilian/master

Changes to Ui messages to avoid appearing bugged

g-lilian committed:
Merge pull request #130 from g-lilian/userguide

Add latest commands to UG

Zhilin-Huang committed:
Merge branch 'master' into dev-guide-studyplan

Zhilin-Huang committed:
Update code formatting

Zhilin-Huang committed:
Update sequence diagrams for get and update study plan

Zhilin-Huang committed:
Update User Guide for study plan related features

Zhilin-Huang committed:
Add Parser component description to Developer Guide

Zhilin-Huang committed:
Merge pull request #121 from Zhilin-Huang/dev-guide-studyplan

Update sequence diagrams for get and update study plan

sliu107 committed:
update the randomized review feature in UG

Zhilin-Huang committed:
Merge pull request #132 from Zhilin-Huang/user-guide-study-plan

Update User Guide for study plan related features

Zhilin-Huang committed:
Update study plan section in user guide

g-lilian committed:
Fix link for Developer Guide

Zhilin-Huang committed:
Merge pull request #134 from Zhilin-Huang/parser-component-dev-guide

Add Parser component description to Developer Guide

quinnyyy committed:
Rework Commands to return a CommandFeedback object. Update tests

quinnyyy committed:
Messed up last commit

quinnyyy committed:
Clean up unused UIs

quinnyyy committed:
Merge branch 'master' into Add-Saving

# Conflicts:
#	.gitignore
#	src/main/java/seedu/tp/Main.java
#	src/main/java/seedu/tp/commands/AddFlashcardToGroupCommand.java
#	src/main/java/seedu/tp/commands/DeleteCommand.java
#	src/main/java/seedu/tp/commands/EventFlashcardCommand.java
#	src/main/java/seedu/tp/commands/GroupCommand.java
#	src/main/java/seedu/tp/commands/HelpCommand.java
#	src/main/java/seedu/tp/commands/ListCommand.java
#	src/main/java/seedu/tp/commands/OtherFlashcardCommand.java
#	src/main/java/seedu/tp/commands/PersonFlashcardCommand.java
#	src/main/java/seedu/tp/commands/ReviewedCommand.java
#	src/main/java/seedu/tp/commands/TimelineCommand.java
#	src/main/java/seedu/tp/group/GroupList.java
#	src/main/java/seedu/tp/parser/Parser.java
#	src/test/java/seedu/tp/commands/TimelineCommandTest.java

Zhilin-Huang committed:
Update build.gradle for v2.0

sliu107 committed:
add ui component to DG

sliu107 committed:
Merge branch 'master' into add-ui-component-to-DG

sliu107 committed:
separate `yes` and `no` in different line

sliu107 committed:
Merge pull request #136 from sliu107/update-randomized-review-feature-in-UG

update the randomized review feature in UG

sliu107 committed:
Merge pull request #137 from Zhilin-Huang/jar-v2.0

Update build.gradle for v2.0

quinnyyy committed:
Fix all the tests

quinnyyy committed:
Fix checkstyle

quinnyyy committed:
Fix .gitignore

quinnyyy committed:
Update textUI and clean some other stuff up

sliu107 committed:
Merge pull request #139 from sliu107/add-ui-component-to-DG

Add ui component to DG

quinnyyy committed:
Merge pull request #101 from quinnyyy/Add-Saving

Add saving

quinnyyy committed:
Change flashcards and groups to save to separate folders

HAOYUN49 commented on docs/DeveloperGuide.md:
```diff
@@ -4,22 +4,194 @@
+
+The structure of the Ui Component is illustrated in the following diagram:
+
+![UiComponentDiagram](images/UiComponent.png)
```
should the relationship between abstract command class and its child command classes be inheritance instead of dependency?

brandoncjh replied:
I think maybe you could use a class diagram, as it would be more comprehensive in explaining the structure of the Ui component rather than an architechture diagram.

Also, the arrows coming out from bottom component with all the commands look very messy pointing to the same exact point and overlap other components. This may lead to misinterpretations especially if you want to add labels/roles to these arrows. Maybe you could instead use the right angle arrows to go around the bottom component boxes and space them out as along the Command box as well.

HAOYUN49 commented on docs/DeveloperGuide.md:
```diff
@@ -4,22 +4,194 @@
+Given below is the Sequence Diagram for interactions between the `Ui` and `FlashcardFactory` components to complete the
+execution of `deleteCommand`.
+
+![UiSequenceDiagram](images/UiSequenceDiagram.png)
```
In the sequence diagram, there is a FlashcardList class that interacts with UI class. However, in the class diagram, there is no FlashcardList class. Should you add it in the class diagram?

brandoncjh replied:
The objects in the diagram are missing activity bars.

HAOYUN49 commented on docs/DeveloperGuide.md:
```diff
@@ -4,22 +4,194 @@
+
+The structure of the Parser Component is illustrated in the following diagram:
+
+![ParserComponentDiagram](images/ParserComponent.png)
```
this class diagram is neat and understandable

brandoncjh replied:
This class diagram is well drawn.

HAOYUN49 commented on docs/DeveloperGuide.md:
```diff
@@ -4,22 +4,194 @@
+
+The following class diagram shows the structures relevant to the Set Priority feature:
+
+![SetPriorityFeatureClassDiagram](images/SetPriorityFeatureClassDiagram.png)
```
should the line between Prioritycommand and UI be dependency instead of association?

brandoncjh replied:
This class diagram is well illustrated and easy to understand.

HAOYUN49 commented on docs/DeveloperGuide.md:
```diff
@@ -4,22 +4,194 @@
+
+The following sequence diagram shows the relevant interactions behind `setPriorityLevel`:
+
+![setPriorityLevelSequenceDiagram](images/setPriorityLevelSequenceDiagram.png)
```
This sequence diagram is quite good. It includes enough information but still very neat and understandable.

brandoncjh replied:
This sequence diagram is well illustrated and easy to understand as well. Maybe the activity bars for an object that is returning a value that has been returned to it could be spaced out a bit (see PriorityCommand returning to Command which returns to Main). However there is nothing wrong with how it is currently drawn.

HAOYUN49 commented on docs/DeveloperGuide.md:
```diff
@@ -4,22 +4,194 @@
+
+The following sequence diagram shows how the   `reviewRandomFlashcards()` operation works:
+
+![randomReviewSequenceDiagram](images/randomReviewSequenceDiagram.png)
```
This sequence diagram is a little bit complicated. Can you simplify it by omitting some less important details?

brandoncjh commented on docs/DeveloperGuide.md:
```diff
@@ -4,22 +4,194 @@
+
+The following sequence diagram shows how the   `reviewRandomFlashcards()` operation works:
+
+![randomReviewSequenceDiagram](images/randomReviewSequenceDiagram.png)
```
This sequence diagram is very comprehensive! However I do feel it is a bit huge. It could be broken down into multiple diagrams to make the information more easily digestable, using ref and sd boxes. For example, the loop box can be encased within a ref.

brandoncjh replied:
Also, the constructors are missing return arrows and activity bars are also missing for commands.

HAOYUN49 reviewed (commented):
Overall is good and neat while there are some small bugs in diagrams. The explanation part of different features is kind of too brief and simple. Maybe some implementation details can be added.

brandoncjh commented on docs/DeveloperGuide.md:
```diff
@@ -4,22 +4,194 @@
+
+The following sequence diagram shows how the `updateStudyPlan` operation works:
+
+![updateStudyPlanSequenceDiagram](images/updateStudyPlanSequenceDiagram.png)
```
This sequence diagram is well drawn.

brandoncjh commented on docs/DeveloperGuide.md:
```diff
@@ -4,22 +4,194 @@
+
+The following sequence diagram shows how the `getStudyPlanList` operation works:
+
+![getStudyPlanListSequenceDiagram](images/getStudyPlanListSequenceDiagram.png)
```
This sequence diagram is well drawn too.

brandoncjh commented on docs/DeveloperGuide.md:
```diff
@@ -4,22 +4,194 @@
+`parseCommand` API call from `Main` class. We use the interaction between the `Parser` and `DeleteCommand` class as
+ an example: 
+
+![ParserAndCommandSequenceDiagram](images/ParserAndCommandSequenceDiagram.png)
```
This sequence diagram is well drawn

brandoncjh reviewed (commented):
Very good diagrams overall, just need to fill in the missing activity bars. Also maybe you could consider adding some colors to the diagrams to aid in differentiating what components the classes refer to?

Zhilin-Huang committed:
Set theme jekyll-theme-cayman

Zhilin-Huang committed:
Add logging to StudyPlanList

Zhilin-Huang committed:
Remove unused ... notation in user guide

Zhilin-Huang committed:
Add Ui line breaks to fix cluttered UI problem

Zhilin-Huang committed:
Update text Ui testing

Zhilin-Huang committed:
Add How to stop entering details guide in UG

Zhilin-Huang committed:
Update FAQ in UG

Zhilin-Huang committed:
Update invalid index response to avoid confusion

Zhilin-Huang committed:
Remove unused Ui methods

Zhilin-Huang committed:
Update headers for flashcard short descriptions

g-lilian committed:
Add summary of basic contributions to PPP

g-lilian committed:
Move PPP to correct folder

Zhilin-Huang committed:
Update UG FAQ

Zhilin-Huang committed:
Update UG RETURN to ENTER/RETURN

Zhilin-Huang committed:
Merge pull request #166 from Zhilin-Huang/ped-fix

Fix bugs found in PE Dry Run

Zhilin-Huang committed:
Implement DeleteStudyPlanCommand

Zhilin-Huang committed:
Update ParserTest for DeleteStudyPlanCommand

g-lilian committed:
Merge pull request #172 from g-lilian/PPP

Add summary of basic contributions to PPP

Zhilin-Huang committed:
Add DeleteStudyPlanCommandTest

Zhilin-Huang committed:
Update text UI testing for delete study plan command

Zhilin-Huang committed:
Update HelpCommand for DeleteStudyPlanCommand

Zhilin-Huang committed:
Update text UI testing

Zhilin-Huang committed:
Remove unnecessary Ui parameter for commands

g-lilian committed:
Add group descriptions to show-groups command

g-lilian committed:
Add ListPriorityCommand

User can type "list-priority PRIORITY_LEVEL" to list flashcards with a particular pl. Added JUnit test, updated text-ui-test as well.

g-lilian committed:
Fix typo in ListPriorityCommandTest

Zhilin-Huang committed:
Update PPP for Zhilin

Zhilin-Huang committed:
Update docs/README.md

Zhilin-Huang committed:
Update User Guide

Zhilin-Huang committed:
Update PPP for Zhilin

Zhilin-Huang committed:
Add links to Zhilin PPP other contribution section

g-lilian committed:
Merge pull request #175 from g-lilian/master

Add ListPriorityCommand, add group descriptions to show-groups

Zhilin-Huang committed:
Merge pull request #176 from Zhilin-Huang/ppp-zhilin

Update Project Portfolio Page - Zhilin

Zhilin-Huang committed:
Merge pull request #173 from Zhilin-Huang/delete-study-plan

Implement DeleteStudyPlanCommand and remove unnecessary Ui parameter for commands

Zhilin-Huang committed:
Revert "Implement DeleteStudyPlanCommand and remove unnecessary Ui parameter for commands"

Zhilin-Huang committed:
Merge pull request #177 from AY1920S2-CS2113-T14-1/revert-173-delete-study-plan

Revert "Implement DeleteStudyPlanCommand and remove unnecessary Ui parameter for commands"

Zhilin-Huang committed:
Implement DeleteStudyPlanCommand and remove unnecessary Ui parameter for commands

Zhilin-Huang committed:
Fix bug in runtest.sh failure

Zhilin-Huang committed:
Merge pull request #178 from Zhilin-Huang/delete-study-plan

Implement DeleteStudyPlanCommand and remove unnecessary Ui parameter …

Zhilin-Huang committed:
Update parser component and study plan features description in Developer Guide

sliu107 committed:
fix the bugs of random commands

sliu107 committed:
add function delete group specified group ID from group list

sliu107 committed:
add white space and delete ui from the fields of delete group command

sliu107 committed:
update add flashcard  to group command to able specify a group by its index

sliu107 committed:
delete some white spaces

sliu107 committed:
refactor the names of some local variables

sliu107 committed:
update an expected feedback

sliu107 committed:
update expected.txt

sliu107 committed:
update epected.txt

sliu107 committed:
update project portfolio page

sliu107 committed:
Merge pull request #184 from sliu107/add-Project-Portfolio-Page

update project portfolio page

g-lilian committed:
Rename PPP and update AboutUs

g-lilian committed:
Add navigator to top of UG

g-lilian committed:
Add delete-plan and list-priority to UG

g-lilian committed:
Minor fixes to UG

g-lilian committed:
Update sequence diagrams

Updated setPriorityLevel sequence diagram and priority feature class diagram

Zhilin-Huang committed:
Use lower resolution diagrams

g-lilian committed:
Add colour to Flashcard

g-lilian committed:
Add colour to Flashcard again

g-lilian committed:
Add colour to Flashcard again

Oops, made a careless mistake just now

sliu107 committed:
rephrase the reentering message

Zhilin-Huang committed:
Scale the ParserAndCommandSequenceDiagram smaller in DG

Zhilin-Huang committed:
Add return messages for study plan related sequence diagrams

Zhilin-Huang committed:
Merge pull request #180 from Zhilin-Huang/dg-study-plan-update

Update parser component and study plan features description in DG

g-lilian committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T14-1/tp

g-lilian committed:
Update DG headers

Updated all feature headers to format ####XXX Feature - Proposed Implementation

g-lilian committed:
Add confirmation message to setPriorityLevel seq diag

quinnyyy committed:
Update stuff for loading

g-lilian committed:
Merge pull request #187 from g-lilian/developerguide

Update priority feature sequence diagrams

g-lilian committed:
Fix intro format in UG

g-lilian committed:
Update help message

Changed the options for possible date formats, removed square brackets from compulsory parameters, added list-priority command.

g-lilian committed:
Fix bug in studyplan deletion message

g-lilian committed:
Prevent users from entering start>end date in flashcard creation

g-lilian committed:
Prevent users from entering start>end in timeline

g-lilian committed:
Fix compilation error

g-lilian committed:
Fix help message in EXPECTED.txt

g-lilian committed:
Update PPP - Lilian

Copied over UG and DG parts for features I implemented.

g-lilian committed:
Fix study plan methods comments

g-lilian committed:
Update text-ui-test

g-lilian committed:
Add hierarchical numbering to UG

sliu107 committed:
update text-ui-test

g-lilian committed:
Merge pull request #188 from g-lilian/master

Fix date-related and message bugs, update PPP

quinnyyy committed:
Add loading

sliu107 committed:
more cases added into text-ui-test

quinnyyy committed:
Fix checkstyle

sliu107 committed:
Merge pull request #181 from sliu107/debug-random-reviewed-command

fix the bugs of random commands

quinnyyy committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T14-1/tp

# Conflicts:
#	src/main/java/seedu/tp/storage/Storage.java

quinnyyy committed:
Get rid of unused variables

g-lilian committed:
Update DG

Updated glossary, included some NFRs, removed Architecture section, added v2.1 user stories. updated headers for appendices.

quinnyyy committed:
Remove delete failed to disk message so CI pass

quinnyyy committed:
Remove unused imports

g-lilian committed:
Merge pull request #190 from g-lilian/master

Updates to UG and DG

Zhilin-Huang committed:
Merge pull request #189 from quinnyyy/master

Load flashcards and groups from disk

sliu107 committed:
Merge branch 'master' into delete-group-command

sliu107 committed:
update text-ui-test

sliu107 committed:
update expected.txt

sliu107 committed:
change a comment

sliu107 committed:
Merge pull request #182 from sliu107/delete-group-command

add function delete group

sliu107 committed:
Merge branch 'master' into update-add-flashcard-to-group-command

Zhilin-Huang committed:
Implement study plan save feature

Zhilin-Huang committed:
Update runtest.sh to remove saved files before every run

Zhilin-Huang committed:
Update build.gradle to v2.1

Zhilin-Huang committed:
Update code formatting

Zhilin-Huang committed:
Update text Ui testing for loading from storage

Zhilin-Huang committed:
Merge branch 'master' into study-plan-save

Zhilin-Huang committed:
Update text Ui testing

Zhilin-Huang committed:
Update .gitignore for ACTUAL2.txt

Zhilin-Huang committed:
Update code formatting

Zhilin-Huang committed:
Rename text-ui-test/EXPECTED2.TXT

Zhilin-Huang committed:
Save flashcard list to storage to preserve order of flashcards

Zhilin-Huang committed:
Add JavaDoc comment for loadFlashcardList

sliu107 committed:
update expected.txt

Zhilin-Huang committed:
Avoid duplicate flashcard names

Zhilin-Huang committed:
Add a group to test for group storage

Zhilin-Huang committed:
Update UG FAQ for storage related questions

Zhilin-Huang committed:
Add Instructions for Manual Testing in DG

Zhilin-Huang committed:
Remove extra ; in RandomCommandTest

Zhilin-Huang committed:
Update PPP for Zhilin

Zhilin-Huang committed:
Add DG and UG extracts to PPP for Zhilin

Zhilin-Huang committed:
Rename variables in loadFlashcardList

Zhilin-Huang committed:
Update UG FAQ

Zhilin-Huang committed:
Update UG to specify start time may not be later than end time

Zhilin-Huang committed:
Add JavaDoc comments to Ui public methods

Zhilin-Huang committed:
Clean up unused Ui methods

Zhilin-Huang committed:
Update JavaDoc comments in StudyPlanList

Zhilin-Huang committed:
Update StudyPlanList

Zhilin-Huang committed:
Update study plan related commands

Zhilin-Huang committed:
Update FindCommand

Zhilin-Huang committed:
Update ListCommand

Zhilin-Huang committed:
Update OtherFlashcardCommand

Zhilin-Huang committed:
Update ListReviewedCommand

Zhilin-Huang committed:
Update ListReviewedCommand

Zhilin-Huang committed:
Update ListReviewedCommand

Zhilin-Huang committed:
Update Flashcard class

Zhilin-Huang committed:
Clean up exceptions

Zhilin-Huang committed:
Update UnknownCommandException

quinnyyy committed:
Merge pull request #191 from Zhilin-Huang/study-plan-save

Implement study plan save feature

quinnyyy committed:
Swallow errors if there is a JSON parse exception when loading files. This makes it so the application does not throw an exception if saved files are corrupted. They are just unused instead

Zhilin-Huang committed:
Update UG study plan section

Zhilin-Huang committed:
Update UG extract in PPP for Zhilin

Zhilin-Huang committed:
Fix formatting in DG Instructions for Manual Testing

Zhilin-Huang committed:
Merge pull request #196 from Zhilin-Huang/ug-update

Minor updates to UG study plan section and fix DG Manual Testing section formatting

quinnyyy committed:
Remove unused import

quinnyyy committed:
Merge pull request #195 from quinnyyy/master

Swallow errors if there is a JSON parse exception when loading files.…

quinnyyy committed:
Implement deletion from group when card is deleted

quinnyyy committed:
Fix checkstyle.

quinnyyy committed:
Fix text-ui test.

quinnyyy committed:
Add Storage Component to DG

g-lilian committed:
Fix navigator links in UG

g-lilian committed:
Rename PPP

Zhilin-Huang committed:
Merge pull request #198 from quinnyyy/Updated-Storage_DG

Updated storage dg

Zhilin-Huang committed:
Merge pull request #199 from g-lilian/master

Fix navigator links in UG

g-lilian committed:
Fix links in PPP

g-lilian committed:
Merge pull request #200 from g-lilian/master

Fix links in PPP

quinnyyy committed:
Add PPP enhancements.

quinnyyy committed:
Add PPP enhancements. Don't know why the last commit didn't work

quinnyyy committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T14-1/tp into Add-PPP

quinnyyy committed:
Add newline.

quinnyyy committed:
Update the user guide with saving and loading information

Zhilin-Huang committed:
Fix UG formatting

Zhilin-Huang committed:
Update Instructions for Manual Testing in DG

Zhilin-Huang committed:
Merge pull request #202 from quinnyyy/Add-Storage-UG

Update the user guide with saving and loading information

Zhilin-Huang committed:
Merge branch 'master' into manual-testing-dg

Zhilin-Huang committed:
Add Saving and Loading section in DG Manual Testing

Zhilin-Huang committed:
Update DG User Stories

Zhilin-Huang committed:
Update DG User Stories

Zhilin-Huang committed:
Fix DG user stories formatting

Zhilin-Huang committed:
Update Instructions for Manual Testing in DG

Zhilin-Huang committed:
Merge pull request #203 from Zhilin-Huang/manual-testing-dg

Update Instructions for Manual Testing in DG

quinnyyy committed:
Add other info to ppp

quinnyyy committed:
Merge pull request #204 from quinnyyy/Add-PPP

Add ppp

Zhilin-Huang committed:
Minor DG format fix

Zhilin-Huang committed:
Merge pull request #205 from Zhilin-Huang/dg-fix

Minor DG format fix

Zhilin-Huang committed:
Update DG and UG extracts in PPP for Zhilin

Zhilin-Huang committed:
Merge branch 'master' into ppp-dg

Zhilin-Huang committed:
Update DG extract in PPP for Zhilin

Zhilin-Huang committed:
Merge pull request #206 from Zhilin-Huang/ppp-dg

Update DG and UG extracts in PPP for Zhilin

Zhilin-Huang committed:
Minor fix for UG formatting

Zhilin-Huang committed:
Merge pull request #207 from Zhilin-Huang/ug-fix

Minor fix for UG formatting

sliu107 committed:
merge master

Zhilin-Huang committed:
Update code formatting

sliu107 committed:
Merge pull request #183 from sliu107/update-add-flashcard-to-group-command

update add flashcard  to group command

Zhilin-Huang committed:
Update PPP for Zhilin

Zhilin-Huang committed:
Merge branch 'master' into ppp-fix

Zhilin-Huang committed:
Update code formatting

Zhilin-Huang committed:
Merge pull request #208 from Zhilin-Huang/ppp-fix

Update PPP for Zhilin

Zhilin-Huang committed:
Update UG study plan

Zhilin-Huang committed:
Update UG extract in Zhilin PPP

Zhilin-Huang committed:
Update Zhilin PPP format

Zhilin-Huang committed:
Fix link in AboutUs.md

Zhilin-Huang committed:
Fix grammar error in UG

Zhilin-Huang committed:
Merge pull request #209 from Zhilin-Huang/ug-update-2.1

Update UG and Zhilin's PPP

Zhilin-Huang committed:
Change back Zhilin PPP format

Zhilin-Huang committed:
Merge pull request #210 from Zhilin-Huang/ug-update-2.1

Change back Zhilin PPP format

g-lilian committed:
Update PPP and replace link for quinn in about-us

Quinn's link was to johndoe.md

g-lilian committed:
Merge branch 'master' into master

g-lilian committed:
Merge pull request #211 from g-lilian/master

Update PPP and replace link for quinn in about-us

Zhilin-Huang committed:
Make Zhilin PPP more concise

Zhilin-Huang committed:
Merge pull request #212 from Zhilin-Huang/concise-ppp

Make Zhilin PPP more concise

sliu107 committed:
Merge branch 'master' into add-Project-Portfolio-Page

sliu107 committed:
update the link name

sliu107 committed:
Merge pull request #213 from sliu107/add-Project-Portfolio-Page

Update link name in PPP

sliu107 committed:
rename my ppp file

sliu107 committed:
Merge pull request #214 from sliu107/master

rename my ppp file
